src/platforms/python/guides/aws-lambda/index.mdx

```diff
@@ -23,7 +23,7 @@ Install our Python SDK using `pip`:
 ```bash
 pip install --upgrade sentry-sdk
 ```
-We also support [installing Sentry in Lambda Layer](/platforms/python/guides/aws-lambda/layer/).
+We also support [installing Sentry in Lambda Layer](/platforms/python/guides/aws-lambda/layer/) and [installing Sentry as a Container Image](/platforms/python/guides/aws-lambda/container-image/).
```

Comment:
Switching the order of these to match the order in which they're listed in the navigation

```suggestion
We also support [installing Sentry as a Container Image](/platforms/python/guides/aws-lambda/container-image/) and [installing Sentry in Lambda Layer](/platforms/python/guides/aws-lambda/layer/).
```